drape/overlay_tree.cpp

```diff
@@ -155,7 +155,7 @@ void OverlayTree::Add(ref_ptr<OverlayHandle> handle)
   }
 
   int const rank = handle->GetOverlayRank();
-  ASSERT_LESS(rank, m_handles.size(), ());
+  ASSERT_LESS(rank, static_cast<int>(m_handles.size()), ());
```

Comment:
Am I right that rank is always non-negative? If so, could you please cast rank to size_t instead of casting size_t to int?